core/client/src/main/java/alluxio/client/file/options/ListStatusOptions.java

```diff
@@ -13,6 +13,7 @@
 
 import alluxio.annotation.PublicApi;
 import alluxio.thrift.ListStatusTOptions;
+import alluxio.thrift.LoadMetadataType;
```

Comment:
(exposing Thrift types to the application layer prevents us from replacing Thrift with other network protocol without breaking backwards compatibility)